react-ui/src/components/Layouts/Dashboard.js

```diff
@@ -62,6 +63,12 @@ class Dashboard extends Component {
           <li>
             <Link to={`/Dashboard/CreateAppointment/${this.state.email}`}>Create Appointment</Link>
           </li>
+
+          <li>
+            <Link to ={`/WorkerAvailableInputPage`}>Worker Input Time Page</Link>
+
```

Comment:
Hi Kyle,
Minor concern but also remove the unneeded whitespace on line 69 please.